### PR TITLE
Add day 8 and 9 energizers. 

### LINKED ISSUE #481: Add day 8 and 9 energizers to playbook
- https://github.com/hackedu/hack-camp/issues/455
- https://github.com/hackedu/hack-camp/issues/469

### PR DESCRIPTION
**Pull request by zachlatta**
@ssun098 can I get your eyes on this? I reworded some of the energizers in #455 and #469 and would love your feedback.

Closes #481.

### CODE REVIEW

**Comment by jonleung** [comment]
@ssun098, warm friendly poke ^_^

**Comment by selynna** [comment]
@jonleung @zachlatta my bad I didn't see this... I'll get to this in an hour or so?

**Comment by selynna** [comment]
@zachlatta I'm going to add some stuff into Pulse Game

**Comment by jonleung** [comment]
@ssun098, great! Please:
1. leave a comment here when you are finished making the changes (or else no one would know that it is ready to be merged)
2. assign someone to this pull request when it is ready to be merged (or it won't be clear who is responsible for merging it and it won't be merged).

@MaxWofford, please correct if the above steps are wrong.

**Comment by selynna** [comment]
Sorry! I didn't realized I could actually edit the file.  Changes made, so I'm assigning @MaxWofford to this, if that's okay?

**Comment by jonleung** [comment]
@ssun098, assigning to @MaxWofford makes sense to me because he is taking lead on making sure the other energizers are added as well.

File: cohort_4/playbook/ACTIVITIES.md [hunk 1]
```diff
@@ -404,6 +415,20 @@ Team sizes:
...
+
+Assemble the group in two lines, holding hands. Everyone should have their eyes
+closed except for the leader of each line. A beach ball or other object is
+placed at the opposite end of lines from the leaders. The facilitator should
+stand by the leaders and flip a coin. If heads, leaders start pulse race by 
+squeezing the hand of the person next to them, which sets off a chain reaction. 
+When the last person in line feels their hand squeezed, they may open their 
+eyes and grab for the beach ball. The team that gets the ball first gets a point 
+for the round If tails, do nothing. If a pulse is set off when the coin is tails 
+and someone grabs the beach ball, the team loses a point. . After each round,
```
Comment (maxwofford) [inline cohort_4/playbook/ACTIVITIES.md]: `team loses a point. . After each`

**Comment by maxwofford** [comment]
Thanks @ssun098!

**Comment by selynna** [comment]
@MaxWofford can we delete the branch?

**Comment by maxwofford** [comment]
@ssun098 Yep! Thanks :)

**Pull request merged**